packages/lesswrong/components/posts/PostsPage/PostCoauthorRequest.tsx

```diff
@@ -67,11 +68,12 @@ const PostCoauthorRequest = ({post, currentUser, classes}: {
 
   const onResponse = async (accept: boolean) => {
     setLoading(true);
-    const { errors } = await acceptCoauthorRequest({variables: {
+    const { errors }: ExecutionResult = await acceptCoauthorRequest({variables: {
```

Comment:
For totally inscrutable reasons, Typescript decided that the return type from `acceptCoauthorRequest` (a `FetchResult`, which extends `ExecutionResult` no longer had an `errors` field (even though, drilling in to the type definitions, it did).  Manually annotating it with `FetchResult` failed.  Manually annotating it with `ExecutionResult` (which is extended by `FetchResult`) works, though that's probably just because it's a record type with two optional fields (which means you could annotate practically anything with it).